cvat-core/src/project.js

```diff
@@ -257,7 +267,14 @@
                 name: this.name,
                 assignee_id: this.assignee ? this.assignee.id : null,
                 bug_tracker: this.bugTracker,
-                labels: [...this.labels.map((el) => el.toJSON())],
+                labels: [
+                    ...this.labels.map((el) => el.toJSON()),
+                    ...this._deletedLabels.map((label) => {
+                        const labelJSON = label.toJSON();
+                        labelJSON.deleted = true;
+                        return labelJSON;
```

Comment:
Should not we keep "deleted" flag in a Label instance as it done for annotations?

Reply:
In this case we probably do not need additional ``data.deleted_labels`` fields in both project and task classes and I think some duplicated code will be removed.